tests/gtests/graph/unit/utils/test_pattern_matcher_cpu.cpp

```diff
@@ -1108,6 +1108,53 @@ TEST(test_utils_pattern_matcher, OptionalWithLargerPort) {
     ASSERT_EQ(fusion_ops.size(), 3U);
 }
 
+TEST(test_utils_pattern_matcher, CommutativeSelect) {
```

Comment:
The commit message shoud be gtest: graph: ...

Reply:
Updated. Just typo...